streams/src/test/java/org/apache/kafka/streams/integration/RocksDBMetricsIntegrationTest.java

```diff
@@ -101,8 +101,9 @@
     @Parameters(name = "{0}")
     public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][] {
+            {StreamsConfig.AT_LEAST_ONCE},
             {StreamsConfig.EXACTLY_ONCE},
-            {StreamsConfig.AT_LEAST_ONCE}
+            {StreamsConfig.EXACTLY_ONCE_BETA}
         });
```

Comment:
General question: we record the same metric for all three processing modes. Why do you need to parametrize this test? \cc @cadonna